### PR TITLE
Add CocoaPods 0.37.2 formula

### PR DESCRIPTION
**Pull request by keith**
See more conversation about this here: https://github.com/CocoaPods/CocoaPods/pull/3010

// @mikemcquaid

### CODE REVIEW

File: Library/Formula/cocoapods.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Cocoapods < Formula
+  homepage "https://github.com/keith/cocoapods/"
```
Comment (keith) [inline Library/Formula/cocoapods.rb]: I'm happy to update this homepage to the main repo if that makes more sense.

File: Library/Formula/cocoapods.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  url "http://keith.github.io/CocoaPods/cocoapods-0.37.2.tar.gz"
+  sha1 "f727a8027a747f01323ee1188c4ce654731e3e51"
+
+  depends_on "xcproj" => :recommended
```
Comment (segiddins) [inline Library/Formula/cocoapods.rb]: Xcproj isn't used any more?

Comment (keith) [inline Library/Formula/cocoapods.rb]: Awesome!

**Comment by mikemcquaid** [comment]
Hey @keith, it seems `brew install Caskroom/cask/cocoapods` works now so I'm wondering if we still need this. What do you think?

**Comment by keith** [comment]
Personally I'm not really interested in dealing with another tool too install a GUI to install a CLI. But if you think that's the way to go I can close this.

**Comment by mikemcquaid** [comment]
In this case it seems to install `pod` into `/usr/local/bin` with the single command. Dunno if that affects things?

**Comment by keith** [comment]
Yea it does. I'm not a huge fan of being dependent on cask and the GUI for that though. Like I said if you think that's the way to go that's fine though.

**Comment by mikemcquaid** [comment]
I think I'll pass on this in favour of that just as it's officially supported upstream. It still annoys me, for what it's worth, though :sob:

**Pull request closed without merging**